src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rentalPrice = 0;
+
+  if (days < 3) {
+    rentalPrice = days * 40;
+  } else if (days >= 3 && days < 7) {
```

Comment:
`days >= 3` will always be `true` here due to previous `if` statement

```suggestion
  } else if (days < 7) {
```

Reply:
Ok, understood, TU!